shared/modules/hexstring-utils.js

```diff
@@ -0,0 +1,31 @@
+import {
+  isHexString,
+  isValidAddress,
+  isValidChecksumAddress,
+  addHexPrefix,
+} from 'ethereumjs-util';
+
+export const BURN_ADDRESS = '0x0000000000000000000000000000000000000000';
+
+export function isBurnAddress(address) {
+  return address === BURN_ADDRESS;
+}
+
+export function isValidHexAddress(possibleAddress, allowNonPrefixed = true) {
```

Comment:
Nit: This might be easier to understand if the second parameter was an options bag.

e.g. `isValidHexAddress(address, { allowNonPrefixed: false })` v.s. `isValidHexAddress(address, false)`

I was a bit confused about what the `false` meant at first, reading through this PR.

Reply:
I did that and added docs 🤦🏻 was maybe rushing too much last week.